packages/ui-library/src/components/tooltip-container/style.css

```diff
@@ -2,16 +2,10 @@
 	.yst-root {
 		.yst-tooltip-container {
 			@apply yst-relative;
-
-			--yst-display-tooltip: none;
-
-			&:hover, &:focus-within {
-				--yst-display-tooltip: inline-block;
-			}
```

Comment:
This variable is used in the tooltip CSS: https://github.com/Yoast/wordpress-seo/blob/57997126d53dfbf372568aa4c27d26bd989a56f4/packages/ui-library/src/elements/tooltip/style.css#L4

Reply:
Removed